### PR TITLE
Handle input VCFs where the (R)AD or (R)AF fields are of Number 1

### PR DESCRIPTION
**Pull request by susannasiebert**
The Number of the (R)AD and (R)AF fields in the input VCF might be 1,
but we are now writing R/AA number fields now which are is-many.
If this is an multi-sample VCF the parsed entries of the other
samples need to be changed from a single number to an array or
else the vcf_writer will throw an error

### CODE REVIEW

**Comment by zlskidmore** [comment]
+1

File: vcf_annotation_tools/vcf_readcount_annotator.py [hunk 1]
```diff
@@ -232,6 +232,22 @@ def main(args_input = sys.argv[1:]):
...
+            other_samples = vcf_reader.header.samples.names
+            other_samples.remove(sample_name)
+            for sample in other_samples:
+                if (count_field in entry.call_for_sample[sample].data and
+                   (not isinstance(entry.call_for_sample[sample].data[count_field], list))):
+                    entry.call_for_sample[sample].data[count_field] = [entry.call_for_sample[sample].data[count_field]]
+                if (frequency_field in entry.call_for_sample[sample].data and
+                   (not isinstance(entry.call_for_sample[sample].data[frequency_field], list))):
+                    entry.call_for_sample[sample].data[frequency_field] = [entry.call_for_sample[sample].data[frequency_field]]
+
```
Comment (tmooney) [inline vcf_annotation_tools/vcf_readcount_annotator.py]: Would it work for these to use something like:
```python
  sample_data = entry.call_for_sample[sample].data
  if (count_field in sample_data and (not isinstance(sample_data[count_field], list))):
    sample_data[count_field] = [sample_data[count_field]]
  if (frequency_field in sample_data and (not isinstance(sample_data[frequency_field], list))):
    sample_data[frequency_field] = [sample_data[frequency_field]]
```
or:
```python
  sample_data = entry.call_for_sample[sample].data
  for field in [count_field, frequency_field]:
    if (field in sample_data and (not isinstance(sample_data[field], list))):
      sample_data[field] = [sample_data[field]]
```

**Review by tmooney — APPROVED** [review]
+1, thanks for figuring this one out! 😄

File: vcf_annotation_tools/vcf_readcount_annotator.py (hunk 1 above)
Comment (susannasiebert) [reply vcf_annotation_tools/vcf_readcount_annotator.py]: yeah, I was just thinking the same thing. I'll make that change.

**Pull request merged**